packages/govuk-frontend-review/src/common/nunjucks/globals/markdown.mjs

```diff
@@ -7,5 +8,14 @@ import { marked } from 'marked'
  * @returns {string} Rendered Markdown
  */
 export function markdown (content) {
+  // mangle is enabled by default, but deprecated since marked 5.0.0.
+  // Explicitly set it to false to avoid deprecation warnings
+  marked.use({
+    mangle: false
+  })
+  // headerIds and headerPrefix are enabled by default, but deprecated since
+  // marked 5.0.0. Use recommended plugin instead
+  marked.use(gfmHeadingId())
```

Comment:
Sudden thought. Do we actually have any headings in `example.scenario` fields?

Maybe we can get away with `{ headerIds: false }`

Reply:
Though it does feel nicer to not install a needless plugin!